### DISCUSSION TITLE
Any way to update cacheControl policies for storage bucket?

### DISCUSSION THREAD

**Discussion by anngbaum** [supabase/supabase#21119] [Questions]
We recently went through a migration to resize our images and then upload the resized version to storage.

However, in this process, we had a miscommunication that resulted in the new uploads including:

`cacheControl: 'no-cache'` (oops!)

I understand that Supabase Storage is just using S3 buckets, and I've looked into the AWS guidance on both [expiration](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html#ExpirationDownloadDist) and controlling the [Cache Key using policies](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/controlling-the-cache-key.html), but didn't see anything that seemed accessible via Supabase. The extent of configuration I was able to find was just object types supported, and maximum file size (but please feel free to point me in the direction if there's more control over our buckets that I wasn't aware of).

Is there any way to bulk update the cacheControl policy for a bucket (such that it overrides the individual policy set per object?) Alternatively, is there any update method on Storage such that I can just update the policy per object rather than re-uploading each object?

Thanks in advance!

**Comment by kyoto21** [comment]
+1
